themes/digital.gov/layouts/shortcodes/do-dont-row.html

```diff
@@ -0,0 +1,10 @@
+{{- if and (.Get "do") (.Get "dont") -}}
```

Comment:
Is this `if and …` a common Hugo pattern? What's it supposed to check for?

Reply:
This checks for if both a `do` and a `dont` exist for the new row, otherwise it won't display the row when one of them is missing.